lib/input/sketch/entities/style/style.dart

```diff
@@ -30,17 +39,52 @@ class Style {
     this.classField,
     this.colorControls,
     this.contextSettings,
-    this.do_objectID,
+    this.UUID,
     this.endMarkerType,
-    this.fills,
-    this.innerShadows,
+    List<Fill> this.fills,
+    List<Fill> this.innerShadows,
     this.miterLimit,
-    this.shadows,
+    List<Fill> this.shadows,
     this.startMarkerType,
     this.windingRule,
-    this.textStyle,
+    TextStyle this.textStyle,
   });
 
   factory Style.fromJson(Map json) => _$StyleFromJson(json);
-  Map toJson() => _$StyleToJson(this);
+  Map<String, dynamic> toJson() => _$StyleToJson(this);
+
+  @override
+  @JsonKey(ignore: true)
+  PBColor backgroundColor;
+
+  @override
+  var boundaryRectangle;
+
+  @override
+  bool isVisible;
+
+  @override
+  String name;
+
+  @override
+  String prototypeNodeUUID;
+
+  @override
+  String type;
+
+  @override
+  void set borders(List _borders) {
+    // TODO: implement borders
+  }
+
+  @override
+  Future<PBIntermediateNode> interpretNode(PBContext currentContext) {
+    // TODO: implement interpretNode
+    throw UnimplementedError();
```

Comment:
This would cause a Runtime Exception if the style comes from Sketch, no? 🤔

Reply:
You are right